src/clp_ffi_py/modules/clp_four_byte_encoder.cpp

```diff
@@ -0,0 +1,103 @@
+#include <clp_ffi_py/Python.hpp>
```

Comment:
Can you add a comment beside this saying it must be included first? It would be pretty easy for someone to reorder these headers by mistake in the future.

Reply:
Every file should have this header to be included, and it will be redundant if we do it in every file. I left a comment in Python.hpp. Do u think it's necessary?

Reply:
I think technically you had two reviews before mine and yet there was still an instance where `Python.hpp` was included after another header file. So I do think it's necessary to highlight the issue with a comment on every inclusion of `Python.hpp` otherwise someone else will make the same mistake, lol. Maybe one day someone will figure out a way to write an automated check for it, but for now a comment is the best we've got.